### DISCUSSION TITLE
PointCloud disappear at certain angles.

### DISCUSSION THREAD

**Discussion by MeyerBuaharon** [isl-org/Open3D#4793] [Q&A]
When i move my pcd in a certain way it disappear, anyone had this problem and can give me some advice?
i use a simple draw function.

https://user-images.githubusercontent.com/40149670/155153584-006b752c-cd14-4e0e-baaa-aeedb985ff45.mp4

**Comment by ssheorey** [comment]
Thanks for reporting. We are working on a fix. See #4842
